examples/predict_houseprice.py

```diff
@@ -0,0 +1,37 @@
+"""
+===================================================
+Price Predicting using pgmpy and BayesianModel
+===================================================
+For simplicity we will consider that the price of the house depends only on Area, Location, Furnishing,
+Crime Rate and Distance from the airport.  And also we will consider that all of these are discrete variables.
+"""
+from pgmpy.models import BayesianModel
+import pandas as pd
+import numpy as np
+data = pd.DataFrame(raw_data, columns=['A', 'C', 'D', 'L', 'F', 'P'])
```

Comment:
@AMAN3003 You could use some existing data sets from kaggle, there was one about house prices.